### PR TITLE
ping: add -O example

### PR DESCRIPTION
**Pull request by sbrl**
When I'm not using `mtr`, I'll often use the `-O` option to see when my server goes down and comes back up again during a reboot.

----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/ping.md [hunk 1]
```diff
@@ -21,3 +21,7 @@
 - Ping host and ring the bell when a packet is received (if your terminal supports it):
 
 `ping -a {{host}}`
+
+- Also display a message if no response was recieved:
```
Comment (pxgamer) [inline pages/common/ping.md]: Just a typo here with `received`.

File: pages/common/ping.md [hunk 2]
```diff
@@ -21,3 +21,7 @@
...
+
+`ping -O {{host}}`
```
Comment (pxgamer) [inline pages/common/ping.md]: Just interested, what OS is this? 🤔 I can't seem to find it in man pages.

Comment (sbrl) [reply pages/common/ping.md]: What OS? I'm using Ubuntu 18.04 Bionic Beaver. Don't know if this works in other OSes, but i'm pretty sure it does - except windows, of course - the flags are completely differerent there.

Comment (pxgamer) [reply pages/common/ping.md]: Hmm. Seems to work fine on my Ubuntu server.

But not on some other Linux versions. Anyway, I think this is pretty useful. :+1:

Comment (sbrl) [reply pages/common/ping.md]: `ping -V` yields `ping utility, iputils-s20161105`.

Comment (sbrl) [reply pages/common/ping.md]: Hrm, that's odd. Maybe it's a Debian/GNU-specific thing or something?

File: pages/common/ping.md (hunk 1 above)
Comment (sbrl) [reply pages/common/ping.md]: Thanks! Fixed

**Comment by thiswind** [comment]
`-O` option not work on macOS Mojave

wx20190210-181722 2x

**Comment by sbrl** [comment]
@thiswind: I see. Because this page is correct for multiple other platforms, we should create a copy in the `osx` folder with the unsupported option removed.

**Comment by thiswind** [comment]
thank you :)

**Comment by thiswind** [comment]
@sbrl thank you :)

**Pull request merged**